Libplanet/Net/Swarm.cs

```diff
@@ -436,7 +439,7 @@ IEnumerator IEnumerable.GetEnumerator()
                 throw new SwarmException("Swarm is already running.");
             }
 
-            if (_iceServers != null)
+            if (_host is null && _iceServers != null)
```

Comment:
For consistency of style:

```suggestion
            if (_host is null && _iceServers is null)
```

Reply:
`_iceServers != null` and `_iceServers is null` seem to be opposite. did you mean `!(_iceServers is null)` ?

Reply:
Ah, it's my fault.  You're right.

```suggestion
            if (_host is null && !(_iceServers is null))
```